core/src/main/java/org/apache/spark/api/java/function/package.scala

```diff
@@ -22,4 +22,4 @@ package org.apache.spark.api.java
  * these interfaces to pass functions to various Java API methods for Spark. Please visit Spark's
  * Java programming guide for more details.
  */
-package object function 
+package object function
```

Comment:
Now. I see what you mean. We are mentioning different one.
Your example is about `carriage return`.
What I meat is `org.scalastyle.file.WhitespaceEndOfLineChecker`. 
In 'src/scala', please choose one scala file and delete the last empty line and run `dev/scalastyle`.
You can see the violation.